charts/rd-location-ref-api/values.yaml

```diff
@@ -3,16 +3,18 @@ java:
   image: 'hmctspublic.azurecr.io/rd/rd-location-ref-api:latest'
   ingressHost: rd-location-ref-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: rd
-  readinessPath: '/health/readiness'
   environment:
     POSTGRES_HOST: rd-location-ref-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com
-    POSTGRES_USERNAME: dbrdlocationref@rd-location-ref-api-postgres-db-{{ .Values.global.environment }}
-    POSTGRES_DB_NAME: dbrdlocationref
+    POSTGRES_USERNAME: locrefdata@rd-location-ref-api-postgres-db-{{ .Values.global.environment }}
```

Comment:
we changed this dbrdlocationref..it will fail load

Reply:
ok . i will change